StudentList/Fragments/StudentListFragment.cs

```diff
@@ -117,18 +132,51 @@ public override bool OnOptionsItemSelected(IMenuItem item)
 
         private void StudentAdapterItemClick(object sender, Student e)
         {
-            this.ShowStudentInfo(e.Id);
+            var editStudentAction = new EditStudentAction(e.Id);
```

Comment:
`StudentEditRequested` is better name for this action, `EditStudentAction` is better for action that actually commits changes (when you press 'Confirm' on edit student screen).